src/index.html

```diff
@@ -17,16 +17,60 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="/bigger-search-input" method="get"
+      class="search__field search__field--bigger"
+      data-qa="big"
+    >
```

Comment:
The form tag is incorrectly self-closed. In HTML, form elements should not be self-closed. Change `<form ... />` to `<form ...>` and ensure it is properly closed with `</form>`.